Source/KSCrash/Recording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -184,13 +184,25 @@ catch(TYPE value)\
 #pragma mark - Public API -
 // ============================================================================
 
+static void initialize()
+{
+    static bool isInitialized = false;
+    if(!isInitialized)
+    {
+        isInitialized = true;
+        kssc_initCursor(&g_stackCursor, NULL, NULL);
```

Comment:
Ah, yes, I didn't mean to call `kssc_initSelfThread(&g_stackCursor, 1);` to init the actual stack trace from `CPPExceptionTerminate()`, I was talking about whether to delay the fallback-init of `kssc_initCursor(&g_stackCursor, NULL, NULL);` to `CPPExceptionTerminate()`, based on detecting that `kssc_initSelfThread`was never called. As opposed to an unconditional initialize from `setEnabled`.